Username for 'https://github.com/gnmearacaun/python-exercises.git': gnmearacaun
Password for 'https://gnmearacaun@github.com/gnmearacaun/python-exercises.git':
aws --profile codecommit-user codecommit credential-helper $@ store: 1: aws: not found
To https://github.com/gnmearacaun/python-exercises.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'https://github.com/gnmearacaun/python-exercises.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
